lib/workers/branch/schedule.ts

```diff
@@ -49,10 +49,10 @@ export function hasValidSchedule(
     }
     if (
       !parsedSchedule.schedules.some(
-        s => s.d !== undefined || s.D || s.t_a !== undefined || s.t_b
+        s => s.M || s.d !== undefined || s.D || s.t_a !== undefined || s.t_b
```

Comment:
I forget - and clearly didn't document - why some checks were `!== undefined` while others were just simple like `s.t_b`. I don't suppose you looked into this while deciding what to do for `s.M` ?

Reply:
I have to admit I feel for cargo-cult here and copied the check applied to `s.D` without real understanding. :grimacing: